pymde/quadratic.py

```diff
@@ -79,25 +97,41 @@ def _spectral(
         k = m
```

Comment:
`k = m + 1` should apply to both branches now (see my other comment regarding the unsupported argument `Y`). Both branches will also use the same line of code to construct the `order` variable (`order = np.argsort(eigenvalues[1:k]))`).